### DISCUSSION TITLE
Node provision failing 4.7.0-0.okd-2021-06-13-090745

### DISCUSSION THREAD

**Discussion by Udbv** [okd-project/okd#737] [Q&A]
**Describe the bug**
Node provision failing with default image.

machine-config-daemon-firstboot.service - Machine Config Daemon Firstboot
```
Daemon Firstboot...
sed: can't read /etc/yum.repos.d/*.repo: No such file or directory
Running captured: rpm-ostree status --json
 Booted osImageURL:  (47.83.202103251640-0)
Installed Ignition binary version: 2.9.0
 failed to compare MachineConfig: error creating machineConfigDiff for comparison: parsing new Ignition config failed with error: failed to parse Ignition config: parsing Ignition config failed: unknown version.Supported spec versions: 2.2, 3.0, 3.1

```
**Version**

4.7.0-0.okd-2021-06-13-090745 deployed to GCP
**How reproducible**

100% reproducible 
**Log bundle**

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive
